apps/api/src/domain/models/term/tokenization/tokenizer.interface.ts

```diff
@@ -0,0 +1,40 @@
+import { LanguageCode } from '@coscrad/api-interfaces';
+
+/**
+ * TODO[naming] This is tough to name. It's meant to be
+ * either a (possibly length > 1) string of roman characters
+ * that represents either one atomic letter in the target language
+ * or punctuation or white space, or else an out-of-alphabet symbol (e.g. in a loan-word)
+ */
+export type AlphabetCharacters = {
+    text: string;
+    isPunctuationOrWhiteSpace: boolean;
+    isOutOfAlphabet: boolean;
+};
+
+export type Token = {
```

Comment:
We should double check the Spacy API and ensure we are staying as consistent with their standard as possible.